linode/linode-cli
# Fix issue displaying missing action errors when `--help` is specified

## 📝 Description

This pull request resolves an issue that would cause a missing action error to not be displayed if `--help` is specified.

e.g.
```shell
linode-cli placement fake-command --help
```

## ✔️ How to Test

The following test steps assume you have pulled down this PR and run `make install`.

### Unit Testing

```shell
make testunit
```

### Manual Testing

1. In your shell, run `linode-cli placement fake-command --help`
2. Ensure an error is displayed. 
3. Run `linode-cli placement fake-command`
4. Ensure an error is displayed.
